src/components/panel/panel.component.tsx

```diff
@@ -43,7 +48,14 @@ const getTitle = (
   return title as string;
 };
 
-export const Panel = ({ closePanel, objectType, uid }: PanelProps) => {
+export const Panel = ({
+  closePanel,
+  objectType,
+  uid,
+  draggedObject,
+  newContentObject,
```

Comment:
whats the difference between `draggedObject` and `newContentObject` ?

Reply:
`draggedObject` is the object currently being dragged. `newContentObject` is the object dropped.